packages/request-node/test/persistTransaction.test.ts

```diff
@@ -76,21 +73,4 @@ describe('persistTransaction', () => {
       .set('Accept', 'application/json')
       .expect(StatusCodes.INTERNAL_SERVER_ERROR);
   });
-
-  it('should catch IPFS timeout error', async () => {
```

Comment:
I decided to remove this test. it's a bit hard to replicate and I feel it has no real value

Reply:
I disagree with this. The Request Node still writes to IPFS but it should fail gracefully if for some reason that write fails.